<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointsWidget</class>
 <widget class="QWidget" name="PointsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PointsWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_pointList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>781</width>
     <height>541</height>
    </rect>
   </property>
   <property name="title">
    <string>Point list</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>566</y>
     <width>849</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_save">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_pointOpt">
   <property name="geometry">
    <rect>
     <x>805</x>
     <y>14</y>
     <width>141</width>
     <height>537</height>
    </rect>
   </property>
   <property name="title">
    <string>Debug</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_goto">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_getpos">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Teach</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_reload">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>ReLoad</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_stop">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_goto</sender>
   <signal>clicked()</signal>
   <receiver>PointsWidget</receiver>
   <slot>onGoto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>863</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_getpos</sender>
   <signal>clicked()</signal>
   <receiver>PointsWidget</receiver>
   <slot>onGetPos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>863</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reload</sender>
   <signal>clicked()</signal>
   <receiver>PointsWidget</receiver>
   <slot>onReLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>880</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_save</sender>
   <signal>clicked()</signal>
   <receiver>PointsWidget</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>PointsWidget</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSave()</slot>
  <slot>onCancel()</slot>
  <slot>onGoto()</slot>
  <slot>onGetPos()</slot>
  <slot>onReLoad()</slot>
 </slots>
</ui>
